REVIEW.md

```diff
@@ -0,0 +1,85 @@
+# Review process
+> This is a general process, a project specific process must be documented in its own repository.
+
+We rely on peer review as much as possible :
+> https://en.wikipedia.org/wiki/Peer_review
+
+Try to be as benevolent as possible when reviewing your peers. 
+Someone theorise *Egoless programming* that fits our overall philosophy quite well.
+> https://en.wikipedia.org/wiki/Egoless_programming
+
+## Branch
+When starting to work on a subject on any project, the first thing to do is creating your own branch.
+The base branch should be `master` and the new branch should reflect the subject you are working on:
+- a new feature must be prefixed by `feat/` ex: `feat/new-contribution-form`
+- a bug fix can be prefiexed by `fix/`
+Or `chore`, `docs` or any other word you see fit!
+
+> We use Conventionnal commits for commit message, and this branch naming is inspired by the keywords use there
+
+## Commits
+For commit messages and we use Conventionnal commits convention.
+Beware, a single commit shouldn't break anything (nor the test, nor the app, nor anythign.
+We should be able to checkout any commits and run the app without any issues.
+
```

Comment:
Also, we try to keep commits small, and only include the smallest amount of change that keeps it atomic. In other words : let’s create commits that only do one thing.